### DISCUSSION TITLE
How are peaks annotated as intronic/exonic/etc?

### DISCUSSION THREAD

**Discussion by Dragonlongzhilin** [GreenleafLab/ArchR#687] [Questions / Documentation]
I want to view the peak of CA9 gene.
The genome position of this gene is chr9: 35673856-35681159
![image](https://user-images.githubusercontent.com/33304280/114262587-5ff6d480-9a13-11eb-9bc6-50a670fd8c61.png)
I found that two peaks fall into this gene and extracted the result from PeakSet matrix.
![image](https://user-images.githubusercontent.com/33304280/114262564-3fc71580-9a13-11eb-9299-75c7d0751656.png)

But there was a peak (chr9:35669360-35669860) annotated as Exonic, but its genome location is upstream of this gene.
I did't kown why？

**Comment by rcorces** [comment]
Please note that questions and feature requests should no longer be submitted as issues. I have migrated your issue to a Discussion topic which is the appropriate place. Only bugs should be reported under issues. This is indicated in the Issue Template which we would appreciate if all users would utilize.

Peak annotations are determined based on your gene annotation. Presumably there is at least one isoform of a gene (not necessarily CA9) in your gene annotation where there is an exon that at least partially overlaps that peak. In this case CA9 is the gene with the closest annotated start site.
